### ISSUE TITLE
python manage.py migrate is giving an error

### ISSUE THREAD

**Issue by AkhilKamath** [OSDLabs/WavesWebsite2016#12]
```
(waves) A:\My stuff\WavesWebsite2016>python manage.py migrate
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "A:\My stuff\waves\lib\site-packages\django\core\management\__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "A:\My stuff\waves\lib\site-packages\django\core\management\__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "A:\My stuff\waves\lib\site-packages\django\core\management\__init__.py", line 190, in fetch_command
    klass = load_command_class(app_name, subcommand)
  File "A:\My stuff\waves\lib\site-packages\django\core\management\__init__.py", line 40, in load_command_class
    module = import_module('%s.management.commands.%s' % (app_name, name))
  File "A:\My stuff\waves\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "A:\My stuff\waves\lib\site-packages\django\core\management\commands\migrate.py", line 19, in <module>
    from django.db.migrations.autodetector import MigrationAutodetector
  File "A:\My stuff\waves\lib\site-packages\django\db\migrations\autodetector.py", line 13, in <module>
    from django.db.migrations.questioner import MigrationQuestioner
  File "A:\My stuff\waves\lib\site-packages\django\db\migrations\questioner.py", line 12, in <module>
    from .loader import MIGRATIONS_MODULE_NAME
  File "A:\My stuff\waves\lib\site-packages\django\db\migrations\loader.py", line 10, in <module>
    from django.db.migrations.recorder import MigrationRecorder
  File "A:\My stuff\waves\lib\site-packages\django\db\migrations\recorder.py", line 9, in <module>
    class MigrationRecorder(object):
  File "A:\My stuff\waves\lib\site-packages\django\db\migrations\recorder.py", line 23, in MigrationRecorder
    class Migration(models.Model):
  File "A:\My stuff\waves\lib\site-packages\django\db\migrations\recorder.py", line 24, in Migration
    app = models.CharField(max_length=255)
  File "A:\My stuff\waves\lib\site-packages\django\db\models\fields\__init__.py", line 1081, in __init__
    super(CharField, self).__init__(*args, **kwargs)
  File "A:\My stuff\waves\lib\site-packages\django\db\models\fields\__init__.py", line 161, in __init__
    self.db_tablespace = db_tablespace or settings.DEFAULT_INDEX_TABLESPACE
  File "A:\My stuff\waves\lib\site-packages\django\conf\__init__.py", line 48, in __getattr__
    self._setup(name)
  File "A:\My stuff\waves\lib\site-packages\django\conf\__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "A:\My stuff\waves\lib\site-packages\django\conf\__init__.py", line 113, in __init__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
```

**Comment by SebastinSanty** [comment]
It shouldn't be coming. This was already addressed in issue #9 . Maybe you have not updated your local repository.

**Comment by AkhilKamath** [comment]
its working. thanks
